specification/maps/data-plane/Weather/preview/1.0/examples/GetStormCurrentLocation.json

```diff
@@ -0,0 +1,5109 @@
+{
+  "parameters": {
+    "geography": "us",
+    "api-version": "1.0",
+    "format": "json",
+    "year": 2021,
+    "basinID": "NP",
+    "governmentID": 2,
```

Comment:
might need to be updated as camel case after updating weather.json

Reply:
Updated